src/modules/did/did.service.ts

```diff
@@ -83,10 +83,9 @@ export class DIDService {
   }
 
   /**
-   * Retrieves a DID Document for a given DID
-   * @param {DID} did DID whose document should be retrieved
-   * @param {boolean} enhanceWithClaims
-   * @returns {IDIDDocument} Resolved DID Document. Null if no Document is not cached.
+   * Retrieves a DID Document for a given DID. Retrieves from blockchain if not chached.
```

Comment:
A minor typo here: Retrieves from blockchain if not `chached`. >> Retrieves from blockchain if not cached.